_posts/2021-11-18-keras-updates/keras-updates.Rmd

```diff
@@ -1,8 +1,10 @@
 ---
-title: "Keras for R is back!"
+title: "Revisiting Keras for R"
 description: > 
-  For a while, it may have seemed that Keras for R was in some undecidable state, like Schrödinger's cat before inspection. It is high time to  correct that impression. Keras for R is back, with two recent releases adding powerful capabilities that considerably lighten previously tedious tasks. 
-  This post provides a high-level overview. Future posts will go into more detail on some of the most helpful new features, as well as dive into the powerful low-level enhancements that make the former possible.
+  It's been a while since this blog featured content about Keras for R, which may have given the impression that the project was dormant. It is high time to correct that impression.
+  Keras for R is better than ever, with two recent releases adding powerful capabilities that considerably lighten previously tedious tasks.
```

Comment:
```suggestion
  It's been a while since this blog featured content about Keras for R, so you might've thought that the project was dormant. It's not! In fact, Keras for R is better than ever, with two recent releases adding powerful capabilities that considerably lighten previously tedious tasks.
```